polkadot/xcm/xcm-builder/src/barriers.rs

```diff
@@ -39,10 +39,13 @@ impl ShouldExecute for TakeWeightCredit {
 		max_weight: Weight,
 		properties: &mut Properties,
 	) -> Result<(), ProcessMessageError> {
-		log::trace!(
+		tracing::trace!(
 			target: "xcm::barriers",
-			"TakeWeightCredit origin: {:?}, instructions: {:?}, max_weight: {:?}, properties: {:?}",
-			_origin, _instructions, max_weight, properties,
+			origin = ?_origin,
+			instructions = ?_instructions,
```

Comment:
since you are now using them in logging, you can just rename the variable names and remove the underscore:

```suggestion
			?origin,
			?instructions,
```